src/libtriton/arch/x86/x8664Cpu.cpp

```diff
@@ -331,7 +331,7 @@ namespace triton {
 
 
       bool x8664Cpu::isSSE(triton::arch::register_e regId) const {
-        return ((regId >= triton::arch::ID_REG_X86_MXCSR && regId <= triton::arch::ID_REG_X86_XMM15) ? true : false);
```

Comment:
I'm not sure about this. I think SSE only extends registers from xmm0 to xmm15? xmm16-31 are introduced from AVX (EVEX)?

* https://en.wikipedia.org/wiki/AVX-512
* https://en.wikipedia.org/wiki/EVEX_prefix